rakelib/utils.rake

```diff
@@ -0,0 +1,32 @@
+class Utils
+  COLUMN_WIDTH = 30
+
+  def self.podspec_version(file = 'SwiftGen')
+    JSON.parse(`bundle exec pod ipc spec SwiftGen/#{file}.podspec`)["version"]
+  end
+
+  def self.plist_version
+    Plist::parse_xml('SwiftGen/Sources/Info.plist')['CFBundleVersion']
+  end
+
+  def self.log_info(label, msg)
```

Comment:
I think we can come up with a name describing that function better. `log_info` sounds to me like a function which will just write something to stdout with some emoji prefix and fancy colors, to tell "Completed" or "Error: …"

Maybe something like `print_status` or similar? (open to better ideas)